### PR TITLE
[editorial] A few random doc fixes

### PR DESCRIPTION
**Pull request by lmolkova**
1. Update `model/README` - add general overview and links on how to work with yaml
2. Update CONTRIBUTING.md -  add deprecated folder into code organization, move policy check higher
3. Other minor fixes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -167,7 +173,21 @@ When creating new markdown files, you should provide the `linkTitle` attribute.
 This is used to generate the navigation bar on the website,
 and will be listed relative to the "parent" document.
 
-### 3. Verify the changes before committing
+### 4. Check new convention
```
Comment (trask) [inline CONTRIBUTING.md:176]: (and need to adjust the numbers below)
```suggestion
### 3. Check new convention
```

File: Makefile [hunk 2]
```diff
@@ -183,7 +183,7 @@ fix-format:
 # Run all checks in order of speed / likely failure.
 # As a last thing, run attribute registry generation and git-diff for differences.
 .PHONY: check
-check: misspell markdownlint check-format markdown-toc compatibility-check markdown-link-check check-policies attribute-registry-generation
+check: misspell markdownlint check-format markdown-toc markdown-link-check check-policies attribute-registry-generation
```
Comment (trask) [inline Makefile:186]: should we remove `compatibility-check` from Makefile?

File: docs/gen-ai/gen-ai-events.md [hunk 3]
```diff
@@ -25,13 +25,8 @@ linkTitle: Generative AI events
 
 GenAI instrumentations MAY capture user inputs sent to the model and responses received from it as [events](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.40.0/specification/logs/api.md#emit-an-event).
 
-<<<<<<< HEAD
-> [!NOTE]
-> Event API is experimental and not yet available in some languages. Check [spec-compliance matrix](https://github.com/open-telemetry/opentelemetry-specification/blob/main/spec-compliance-matrix.md#events) to see the implementation status in corresponding language.
-=======
 > Note:
 > Event API is experimental and not yet available in some languages. Check [spec-compliance matrix](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.40.0/spec-compliance-matrix.md#logs) to see the implementation status in corresponding language.
->>>>>>> 75746bc9 (reword)
```
Comment (trask) [inline docs/gen-ai/gen-ai-events.md:34]: merge conflict

File: model/README.md [hunk 4]
```diff
@@ -21,18 +23,36 @@ A schema file for VS code is configured in the `/.vscode/settings.json` of this
...
-make table-generation
+make attribute-registry-generation table-generation
 ```
 
 For more information, see the [Weaver](https://github.com/open-telemetry/weaver)
 as our code generations tool.
 
+## Validating semantic conventions
+
+Semantic conventions YAML files are validated by `check-policies` make target for backward compatibility,
```
Comment (trask) [inline model/README.md:47]:
```suggestion
Semantic conventions YAML files are validated by the `check-policies` make target for backward compatibility,
```

**Review by trask — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (lmolkova) [reply Makefile:186]: it's based on the build-tools check and it's duplicating weaver-based backcompat checks in rego files.

We keep it as optional CI check and it helps find some missing policies like https://github.com/open-telemetry/semantic-conventions/issues/1686 or gaps in weaver. 

I don't think anyone needs to run it on their machine though

File: docs/gen-ai/gen-ai-events.md (hunk 3 above)
Comment (lmolkova) [reply docs/gen-ai/gen-ai-events.md:34]: yep, removed it here

**Review by joaopgrassi — APPROVED** [review]

File: docs/general/semantic-convention-groups.md
Comment (chalin) [inline docs/general/semantic-convention-groups.md:1]: When `group-stability.md` was renamed to `docs/general/semantic-convention-groups.md`, then the alias needed to be changed to `group-stability`.

Comment (chalin) [reply docs/general/semantic-convention-groups.md:1]: See:

- #1816

**Pull request merged**